app/Repositories/UserRepository.php

```diff
@@ -41,4 +41,43 @@ public function create($username, $email, $password)
 
     }
 
+    /**
+     * Updates a user on the panel. Returns true if the update was successful.
+     *
+     * @param  string $username
+     * @param  string $email
+     * @param  string $password An unhashed version of the user's password.
+     * @return integer
+     */
+    public function update($id, $email, $isRoot, $password = null)
```

Comment:
This might be possible to do as something like:

```php
public function update($id, $object)
{
    if (array_key_exists('password', $object)) {
        $object['password'] = Hash::make($object['password']);
    }
    User::findOrFail($id)->update($object);
}
```

That would make it so you don't have to pass specific parameters every time, and when it comes time for the API, it will work well for a `PUT/PATCH` request which would only send parameters that need changing.